### ISSUE TITLE
Can't reach 3d models via keyboard navigation

### ISSUE THREAD

**Issue by Floppy** [manyfold3d/manyfold#2105]
You can’t reach the 3D models when using the keyboard. By using ‘tab’ on your keyboard, you should be able to cycle through all interactive elements on the page.

See https://www.w3.org/WAI/WCAG21/Understanding/keyboard

**Comment by Floppy** [comment]
i think in order to resolve this I need to move things back to their own canvas which can actually get focus, and therefore will probably need to do #2071 first.
